automation/conformance.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export WORKSPACE="${WORKSPACE:-$PWD}"
+readonly ARTIFACTS_PATH="${ARTIFACTS-$WORKSPACE/exported-artifacts}"
+mkdir -p "$ARTIFACTS_PATH"
+
+trap "{ make cluster-down; cp -r _out/artifacts/conformance ${ARTIFACTS_PATH}; }" EXIT SIGINT SIGTERM
```

Comment:
SIGTERM isnt needed because we have EXIT, right ?

Reply:
I don't know. Any source?

Reply:
Quique raised it once, 
according this unformal link, it depends on the running shell,
https://unix.stackexchange.com/questions/57940/trap-int-term-exit-really-necessary
you can either try it, leave it as it, or we can find an official source. not crucial.

Reply:
One of the comments:

```
dash also doesn't trap on just EXIT when it receives SIGINT/SIGTERM
```

We use dash on Prow.

Reply:
you rest your case ;]

Reply:
you rest your case ;]
We will have this problem on more scripts, where SIGTERM was omitted

Reply:
btw do you want to add SIGQUIT ? 
same as SIGINT but with core dump

Reply:
one note 
in case `make cluster-down` fails, its error code would mask the real error code in case the script failed
in case its important for you

Reply:
I don't care which RC will be returned as long as it is a failure.

I don't think we need to to work with the expectation of code dump. Do you see a realistic scenario where core dump would happen and it would make sense (or be possible) to execute the trap?

Reply:
just if user triggered 

SIGQUIT is the dump core signal. The terminal sends it to the foreground process when the user presses ctrl-\. The default behavior is to terminate the process and dump core, but it can be caught or ignored. The intention is to provide a mechanism for the user to abort the process. You can look at SIGINT as "user-initiated happy termination" and SIGQUIT as "user-initiated unhappy termination."

I think if we would have a common agreed trap line it would save us time, but up to you.

Btw see PR 428, where there was a shebang that enforce it to run with bash if you want, and then you might can remove the SIGTERM, but its up to you.
(not sure if every node has bash and so on, in case it doesnt, then it wont be good to add it of course)

Reply:
If somebody wants to stop the process by force, why would we gracefully cleanup?

Reply:
both SIGINT and SIGQUIT are catchable, same reason you added SIGINT imo
the cleanup should be done (unless you consider it unhappy as the text said)

just in case he SIGKILL we cant and shouldnt clean